packages/core/src/sorting/sort-participants.ts

```diff
@@ -0,0 +1,62 @@
+import { LocalParticipant, Participant } from 'livekit-client';
+
+/**
+ * Default sort for participants, it'll order participants by:
+ * 1. local participant
+ * 2. dominant speaker (speaker with the loudest audio level)
+ * 3. other speakers that are recently active
+ * 4. participants with video on
+ * 5. by joinedAt
+ */
+export function sortParticipantsByVolume(participants: Participant[]): Participant[] {
```

Comment:
is there a reason why this isn't using the base-sort-functions but rather implementing them again?

Reply:
Good catch. No, I forgot to port it back to this function.